### PR TITLE
Add Azure Private MEC Site Overview workbook

### PR DESCRIPTION
**Pull request by awlodge**
New gallery for Azure Private MEC workbooks, and new workbook for the PMEC site overview.

[Azure Private MEC](https://azure.microsoft.com/en-us/solutions/private-multi-access-edge-compute-mec) combines network functions—including 4G and 5G—applications, and edge-optimized Azure services to deliver high performance, ultra-low-latency solutions that address the modern business needs of enterprise customers. This workbook provides a single-pane view of the status of a Private MEC customer's edge site deployment, including the Azure Private 5G Core deployment and the Azure Stack Edge platform it runs on.

This is the first version of the workbook, and the first workbook template our team has produced. We'd like to make it accessible to our customers to gather feedback and iterate on it based on what customers find useful.

This is my first PR to this repository, so please let me know if I have missed anything.

## Screenshots:
image
image
image
image
image
image

## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.
* [x] Validate your changes using one or more of the [testing](../Documentation/Testing.md) methods.

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [x] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -168,6 +170,7 @@
 /gallery/workbook/microsoft.azurepercept-accounts.json @microsoft/azure-percept-workbooks
 /gallery/workbook/microsoft.database-watcher.json @microsoft/sqlexternalmonitoring
 /gallery/apimanagement-service/** @microsoft/azure-api-management
+/gallery/private-mec/** @microsoft/azure-private-mec-workbooks
```
Comment (gardnerjr) [inline CODEOWNERS:173]: unless we've creaetd a custom workbook "Type" for you (would require information from you about strings and help and support locations and things), this should just be "workbook" and your gallery file should be in the workbook folder, not a new type specific folder.

Comment (awlodge) [reply CODEOWNERS]: Fixed in 03d2fbcdea790b7ca149a6b83d252cb94dea5350 - moved the file into the workbook folder and updated CODEOWNERS to match.

**Review by gardnerjr — APPROVED** [review]
only looking for "technical" accuracy around the gallery, etc.
i don't have any resources of this type to try to test it out, so the "RP specific content and behavior" would need to be validated by your team.

**Comment by awlodge** [comment]
Have now verified the gallery configuration is correct (following fix from workbooks team to support nested resource types):
image

Content of workbook has been validated by our team - though we expect to modify it further based on feedback. Will raise new PRs (which our team can approve) for these modifications).

**Pull request merged**